README.md

```diff
@@ -396,18 +397,56 @@ authClient.signIn({
+
+XHR:
+
+* Executes in the background. The user will see not any change to `window.location`.
+* Will fail if 3rd-party cookies are blocked by the browser.
+* If the session is already closed, the method will fail and reject the returned Promise. This error can be caught by the app and handled seamlessly.
+* It is recommended (but not required) for the app to call `window.location.reload()` after the `XHR` method completes to ensure your app is properly re-initialized in an unauthenticated state.
+* See [Close Current Session](https://developer.okta.com/docs/reference/api/sessions/#close-current-session) in the Session API documentation.
+
+Redirect:
+
+* Will change the `window.location` to an Okta-hosted page before redirecting to a URI of your choice.
+* No issue with 3rd-party cookies.
+* Requires a `postLogoutRedirectUri` to be specified. This URI must be whitelisted in the Okta application's settings.
+* Requires a valid `idToken`. If the `idToken` is invalid or session does not exist the redirect may end on a 400 error page from Okta. This error will be visible to the user and cannot be handled by the app.
+* See [Logout](https://developer.okta.com/docs/reference/api/oidc/#logout) in the OIDC API documentation.
+
+`signOut` takes the following options:
+
+* `postLogoutRedirectUri` - Setting a value will enable the logout redirect method. **The URI must be whitelisted** as a `Logout Redirect Uri` in your application's general settings. It is common to use your application's origin URI as the `postLogoutRedirectUri` so that users are sent to the application "home page" after signout.
+* `state` - An optional value, used along with `postLogoutRedirectUri`. If set, this value will be returned as a query parameter during the redirect to the `postLogoutRedirectUri`
+* `idToken` - Specifies the idToken object. This is only relevant when a `postLogoutRedirectUri` has been specified. By default, `signOut` will look for a token object named `idToken` within the `TokenManager`. If you have stored the idToken object in a different location, you should retrieve it first and then pass it here. In the case where the `idToken` is not available, `signOut` will fallback to using the `XHR` method and then redirect to the `postLogoutRedirectUri`
 
 ```javascript
+// Sign out using the XHR method
 authClient.signOut()
 .then(function() {
   console.log('successfully logged out');
 })
-.fail(function(err) {
+.catch(function(err) {
   console.error(err);
+})
+.then(function() {
+  // Reload app in unauthenticated state
+  window.location.reload();
 });
 ```
 
+```javascript
+// Sign out and redirect to back to the application home page. This URI must be whitelisted in Okta app settings
+authClient.signOut({ postLogoutRedirectUri: window.location.origin });
+```
+
+```javascript
+// In this case, idToken is stored in a custom location
```

Comment:
Suggest,
`// In this case, idToken is stored in a custom location`
->
`// In this case, idToken is stored under the 'myIdToken' key`